requirements/base.txt

```diff
@@ -1,8 +1,255 @@
-selenium==2.39.0
```

Comment:
I'm guessing this is going here because we'll be cleaning this up in a separate ticket?